taxonomies/food/ingredients.txt

```diff
@@ -68,10 +70,6 @@ synonyms:en: semi-wholemeal, semi-whole, semi-whole grain, semi-wholegrain, semi
 
 synonyms:en: semi-sweet, semisweet
 
-synonyms:es: zumo, jugo
-
-synonyms:es: jarabe, sirope
```

Comment:
why did you remove this synonym and the ones below?

for zumo / jugo, we have it as an ingredient en: juice, but for en: syrup we don't have sirope as synonym of jarabe

Reply:
That was the error:

ERROR - duplicate_synonym - 69571 - es:jarabe already is associated to es:jarabe (ingredients) - es:jarabe cannot be mapped to entry en:syrup